server/src/routes/token.ts

```diff
@@ -1,27 +1,19 @@
 import { Request, RequestHandler, Response } from 'express';
 import { jwt } from 'twilio';
 import { TwilioCredentials } from '../common/types';
-import { retrieveAuthentication } from '../middlewares/sample-auth';
 
 export function createTokenRoute(
-  twilioCredentials: TwilioCredentials
+  twilioCredentials: TwilioCredentials,
 ): RequestHandler {
-  const { AccessToken, AccessToken: { VoiceGrant } } = jwt;
+  const {
+    AccessToken,
+    AccessToken: { VoiceGrant },
+  } = jwt;
   return function tokenRoute(_req: Request, res: Response) {
-    const authentication = retrieveAuthentication(res);
-
-    if (typeof authentication === 'undefined') {
-      res.status(403).send('Unauthenticated request.');
-      return;
-    }
-
     const accessToken = new AccessToken(
       twilioCredentials.ACCOUNT_SID,
       twilioCredentials.API_KEY_SID,
       twilioCredentials.API_KEY_SECRET,
-      {
-        identity: authentication.username,
```

Comment:
Note for future reference, we need this for incoming but are removing it presently just for finishing outgoing.

Reply:
Should I just comment it out and leave a note here @mhuynh5757 ?

Reply:
Nah, I think it's ok to remove it. We don't know how the final API will end up looking, i.e. what if we have a different route for getting tokens for incoming calls.